browser/admin/src/Util.js

```diff
@@ -22,7 +22,7 @@ var Util = Base.extend({
 }, { // class interface
 
 	humanizeMem: function (kbytes) {
-		var unit = 1000;
+		var unit = 1024;
 		var units = [_('kB'), _('MB'), _('GB'), _('TB'), _('PB'), _('EB'), _('ZB'), _('YB'), _('BB')];
```

Comment:
I'm always a bit confused here, but isn't it like "KB" is 1000 as a multiplier and "KiB" is 1024 as a multiplier? See the table at https://en.wikipedia.org/wiki/Gigabyte

If I'm right, then 1000 was correct here.

Reply:
OK, let's be consistent an use 1024 everywhere. If people complain, we can reiterate.

Reply:
I think people expect memory sizes in `MiB` and `GiB`, as RAM memory is marketed with binary units (in contrast to other computer memory). It's also what most operating system GUIs use for "file size".

So IMHO `1024` is the correct divisor here. To be more precice, `units` could be changed to `KiB`, `MiB`, ...